contents/blog/transparent-enterprise-pricing.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2022-05-10
+title: Transparent Enterprise Pricing
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog"]
+author: ["simon-fisher"]
+---
+
+[Contact sales](/signup/self-host/get-in-touch?plan=enterprise#contact).  That's what you're usually encouraged to do when you're trying to figure out how much it's going to cost you
+to subscribe to the premium variant of a product.  Then, if you're deemed worthy enough for a sales rep to call you, it's
+highly likely that you won't find out the pricing until you've answered a _lot_ of questions.  And maybe seen a demo.
+And perhaps your own hands-on evaluation.  And also introduced your bosses boss.  Surely there has to be a better way.
+
+##  Why did we implement transparent pricing?
+
+At PostHog we pride ourselves on transparency.  We make our processes, policies, and team structure available for anyone
```

Comment:
It's always best to assume the reader doesn't know anything about us, so I think we should expand this section a little cover the following:

- What is PostHog?
- How is it different to Amplitude etc.?
- A brief overview of the different tiers of the product currently

This will add a little more context into the rest of the post. No need to go super in-depth here. An additional paragraph should suffice.

Reply:
Added an intro section in the latest commit @andyvan-ph